File: src/main/java/net/superricky/tpaplusplus/util/manager/saved/PlayerData.java
package net.superricky.tpaplusplus.util.manager.saved;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

public class PlayerData {
    private boolean tpToggle = false;
    private final List<UUID> blockedPlayers = new ArrayList<>();

    public boolean getTPToggle() {
        return tpToggle;
    }

    public void setTPToggle(boolean tpToggle) {
        this.tpToggle = tpToggle;
    }

    public List<UUID> getBlockedPlayers() {
        return blockedPlayers;
    }
}

File: src/main/kotlin/net/superricky/tpaplusplus/database/PlayerData.kt
package net.superricky.tpaplusplus.database

import com.google.gson.annotations.SerializedName
import java.util.*

class PlayerData(
    @SerializedName("block_players")
    val blockPlayers: MutableSet<UUID> = Collections.synchronizedSet(HashSet()),
    @SerializedName("last_death_pos")
    val lastDeathPos: DeathPos = DeathPos(),
    @SerializedName("toggle")
    var toggle: Boolean = false
)
